### ISSUE TITLE
Journey Card shows incorrect label in journeys page

### ISSUE THREAD

**Issue by richardolsson** [zetkin/app.zetkin.org#657]
## Description
The journey cards on /organize/1/journeys display the `singular_label` when they should display the `title` of each journey.

## Steps to reproduce
1. Create a journey with different `singular_label` and `title` values
2. Go to /organize/1/journeys

To create a journey (if one does not exist) use the following API call (e.g. in the browser console):

```ts
fetch(
  '/api/orgs/1/journeys',
  {
    method: 'POST',
    headers: {'Content-Type': 'application/json'},
    body: JSON.stringify({
      title: 'New member on-boarding',
      plural_label: 'On-boardings',
      singular_label: 'On-boarding'
    })
  }
);
```

### Expected Behaviour
Should show the title of the journey created in 1 ("New member on-boarding" from the example above)

### Actual Behaviour
Instead shows the `singular_label` ("On-boarding").

## Screenshots (if you have any)
![image](https://user-images.githubusercontent.com/550212/168239274-675a944f-dfef-4e87-98b0-ce9160db382e.png)

**Comment by richardolsson** [comment]
I have created another journey type in the dummy, using the `fetch()` mentioned above, so it should not be necessary to do that to reproduce, but instead just visit [/organize/1/journeys](http://localhost:3000/organize/1/journeys).

**Comment by richardolsson** [comment]
The same is true for the close and reopen buttons currently being implemented in #662

**Comment by djbusstop** [comment]
Closing this as it's been fixed in https://github.com/zetkin/app.zetkin.org/pull/699.

I'm pretty sure the related issue regarding the close and reopen buttons depends backend task to extend the journey object so that is planned to be fixed elsewhere.
